readthedocs/core/mixins.py

```diff
@@ -17,3 +17,8 @@ def get_context_data(self, **kwargs):
 class PrivateViewMixin(LoginRequiredMixin):
 
     pass
+
+class AuthenticatedClassesMixin():
```

Comment:
I think this should be named "ProxiedAPIMixin".

Reply:
okay, making this change

Reply:
@stsewd I've fixed this